submissions/benjaminKamena/PHP/unitCoverter.php

```diff
@@ -0,0 +1,20 @@
+// unit conversions
+
+<?php
+
+ function unitCoverter($unit){
```

Comment:
the name of your parameter is confusing. when I saw `$unit` i didn't know you were referring to a value, not until I saw the argument that was passed to it